packages/api-explorer/__tests__/Doc.test.jsx

```diff
@@ -370,3 +371,14 @@ describe('stripSlash', () => {
     expect(operation.path).toBe('/store/inventory/')
   })
 })
+
+test('expand renderCodeAndResponse should be render correctly', () => {
+  const element = mountWithIntl(<Doc {...props} />)
+  expect(element.find('.expandable').find(Button).find(FormattedMessage).prop('id')).toEqual('doc.expand')
```

Comment:
Create a constant `element.find('.expandable')` instead of repeat it

Reply:
if one a constant does not update the rendered tree

Reply:
You could use a constant before the simulate annd one agter the simulate

Reply:
it works! Great